testharness/src/test/java/bio/terra/policy/service/policy/MutatorTest.java

```diff
@@ -34,6 +35,8 @@ void groupConstraintTest() throws Exception {
     var mcgroupPolicy = new PolicyInput(TERRA, GROUP_CONSTRAINT, buildMultimap(GROUP, MCGROUP));
     var mcyugroupPolicy =
         new PolicyInput(TERRA, GROUP_CONSTRAINT, buildMultimap(GROUP, MCGROUP, YUGROUP));
+    var mcyuddgroupPolicy =
+        new PolicyInput(TERRA, GROUP_CONSTRAINT, buildMultimap(GROUP, MCGROUP, YUGROUP, DDGROUP));
```

Comment:
Unrelated to this PR - Interesting, I didn't realize policies are combined. Just curious, why combine, vs having 3 separate group PolicyInputs?

Reply:
It is easier to do the processing if I can use a set and collect all of the relevant input together.
If it turns out to be easier, we can return it as separate policy inputs through the API.